TECHMANIA/Assets/Scripts/Components/Game Scene/Game.cs

```diff
@@ -2084,7 +2085,15 @@ private void OnKeyDownOnLane(int lane)
         {
             // The keystroke is too early or too late
             // for this note. Ignore.
-            PlayKeysound(earliestNote, emptyHit: true);
+            if (!((IList) new NoteType[]{
+                    NoteType.Hold,
+                    NoteType.Drag,
+                    NoteType.RepeatHeadHold,
+                    NoteType.RepeatHold
+                }).Contains(earliestNote.note.type))
+            {
+                PlayKeysound(earliestNote, emptyHit: true);
+            }
```

Comment:
We can avoid duplication of this code by moving this logic inside `PlayKeysound` and checking note type when `emptyHit` is true.